pkg/cmd/server/start/start_node.go

```diff
@@ -280,8 +274,13 @@ func StartNode(nodeConfig configapi.NodeConfig) error {
 
 	config.EnsureVolumeDir()
 	config.EnsureDocker(docker.NewHelper())
+	plugin, endpointFilter := InitSDNPlugin(config, nodeConfig)
```

Comment:
> it's a staged progression, transforming serialized config into built objects, then starting them

@liggitt Fair enough; though still doesn't answer the question about whether NewPlugin() should go into BuildKubernetesNodeConfig() or not.  I think it logically fits there, but to do that we have to move FilteringEndpointsConfigHandler somewhere else to avoid the import cycle between osdn factory and origin kubernetes.